.github/workflows/lint_test_build.yml

```diff
@@ -56,6 +71,11 @@ jobs:
   connector:
     runs-on: ubuntu-latest
     needs: checkout
+    services:
+      redis:
+        image: redis
+        ports:
+          - 6380:6379
```

Comment:
Why not use default ports (likewise for postgres in the accounts service)?

Reply:
That's their current defaults to avoid conflicting with the backend's services
https://github.com/coilhq/rafiki/pull/18/commits/bb305583fff827ef9100c4cd86502f95bc6598cd
I have a  branch with `testcontainers` in the works which will make it moot.
https://github.com/wilsonianb/rafiki/pull/4/files